docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,429 @@
+
+Observability is commonly done using a combination of:
+
+- Logs - record individual operations, such as an incoming request, a failure in a specific component, or an order being placed.
+- Metrics - measuring counters and gauges such as number of completed requests, active requests, widgets that have been sold; or a histogram of the request latency.
+- Distributed tracing - track requests and activities across components in a distributed system so that you can see where time is spent and track down specific failures.
+
+Together, logs, metrics, and distributed tracing are known as the *3 pillars of observability*.
+
+Each pillar might include telemetry data from:
+
+- The .NET runtime, such as the garbage collector or JIT.
+- Libraries, such as from Kestrel (the ASP.NET web server) and `HttpClient`.
+- Application-specific telemetry that's fired by your code.
+
+## Observability approaches in .NET
+
+There are a few different ways to achieve observability in .NET applications:
+
+- Explicitly in code, by referencing and using an SDK such as OpenTelemetry. If you have access to the source code and can rebuild the app, then this is the most powerful and configurable mechanism.
+- Out-of-process using [EventPipe](./eventpipe.md). Tools such as [dotnet-monitor](./dotnet-monitor.md) can listen to logs and metrics and then process them without affecting any code.
+- [Using a startup hook](https://github.com/dotnet/runtime/blob/main/docs/design/features/host-startup-hook.md), assemblies can be injected into the process that can then collect instrumentation. This mechanism enables creating instrumentation for libraries that weren't designed for it but can be fragile to version conflicts between application and instrumentation dependencies. An example of this approach is [OpenTelemetry .NET Automatic Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/README.md).
+
+## What is OpenTelemetry
+
+[OpenTelemetry](https://opentelemetry.io/) (OTel) is a cross-platform, open standard for collecting and emitting telemetry data. OpenTelemetry includes:
+
+- [APIs](https://opentelemetry.io/docs/concepts/instrumentation/manual/) for libraries to use to emit telemetry
+- [Semantic conventions](https://github.com/open-telemetry/semantic-conventions) for metrics & traces
+- Standard for [exporter components](https://opentelemetry.io/docs/collector/)
+- [OTLP wire protocol](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/README.md) for transmitting telemetry data
+
+There are OpenTelemetry implementations for most languages and platforms, including .NET.
+
+## .NET Observability architecture
+
+The .NET OpenTelemetry implementation is a little different from other platforms, as .NET provides logging, metrics, and activity APIs in the BCL. That means OTel doesn't need to provide APIs for library authors to use. The .NET implementation uses these existing APIs for instrumentation:
```

Comment:
Two suggestions:
1. maybe reword BCL considering many readers might not know what that means.
2. instead of saying ".NET uses existing APIs", use ".NET and OpenTelemetry community have been working closely ..." (feel free to steal some wording from https://github.com/dotnet/aspnetcore/issues/48309#issuecomment-1558631599 and rephrase it)